lib/dash/content_protection.js

```diff
@@ -276,7 +357,19 @@ shaka.dash.ContentProtection.convertElements_ = function(
           goog.asserts.assert(!element.init || element.init.length,
                               'Init data must be null or non-empty.');
           var initData = element.init || defaultInit;
-          return [ManifestParserUtils.createDrmInfo(keySystem, initData)];
+          var drmInfo = ManifestParserUtils.createDrmInfo(keySystem, initData);
+
+          // extract Widevine license URL
+          if (keySystem === 'com.widevine.alpha') {
```

Comment:
Is this key system only used for the microsoft specific widevine?

Reply:
This will appear for the Widevine pssh node as well

Reply:
is having the empty string for the `licenseServerUri` what is desired in the pssh node case?

Reply:
Yes